packages/snjs/test/lib/003.encryption.test.ts

```diff
@@ -1,37 +1,25 @@
 import {
   Platform,
   Environment,
-  DeinitSource,
   KeyParamsOrigination,
   SNProtocolOperator003
 } from '@Lib/index';
-import { createApplication } from '../setup/snjs/appFactory';
+import { createApplication } from '../factory';
 import SNCrypto from '../setup/snjs/snCrypto';
 
 describe('Test 003 encryption', () => {
-  /** The global Standard Notes application. */
-  let testSNApp;
-
   const identifier = 'hello-test@sn.org';
   const password = 'this-is-a-password-1234';
-  let rootKey;
-
   const protocol003 = new SNProtocolOperator003(new SNCrypto());
 
-  beforeEach(async () => {
-    testSNApp = await createApplication('test-application', Environment.Web, Platform.LinuxWeb);
-    rootKey = await protocol003.createRootKey(
+  it('should properly encrypt and decrypt a piece of text', async () => {
+    createApplication('test-application', Environment.Web, Platform.LinuxWeb);
```

Comment:
Doesn't seem like this application is necessary? If it is we need to de-init it.

Reply:
The `application` is needed for the `Uuid` functions. Will de-init at the end of the test.